js/scripts.js

```diff
@@ -409,6 +411,17 @@ function debugInstitutionLogos() {
     }
 }
 
+function debugInstitutionLogosImageExistence() {
+    // Check to ensure the URL to each institution's logo image hasn't moved, died, etc.
+    for (let image of logos) {
+        imageExists(image[1], exists => {
+            if (!exists) {
+                console.log(`Link for the logo of ${image[0]} does not return a valid image. URL: ${image[1]}`)
+            }
+        });
+    }
```

Comment:
`image[0]` and `image[1]` are a little ambiguous and require someone to go find out what those properties are.
```suggestion
    for (const [ destination, url ] of logos) {
        imageExists(url, exists => {
            if (!exists) {
                console.log(`Link for the logo of ${destination} does not return a valid image. URL: ${url}`)
            }
        });
    }
```